defaults/main.yml

```diff
@@ -1,4 +1,4 @@
-ferm_rules_defaults:
```

Comment:
we expect user to override ferm_rules variables, by doing so if 'mergeDict' option isn't set to true, user will lose the defaults setup.

This is why there is 2 differents values, it came from a previous PR.